tests/mocks/ERC721_SafeMintable_mock.cairo

```diff
@@ -209,17 +192,17 @@ func safeMint{
         data: felt*
     ):
     Ownable_only_owner()
-    ERC721_safeMint(to, tokenId, data_len, data)
+    ERC721._safe_mint(to, tokenId, data_len, data)
```

Comment:
This `tokenId, data_len` bit is the most annoying code in the library. It's the next thing to tackle.